smart_proxy_dynflow.gemspec

```diff
@@ -21,7 +21,10 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '~> 2.5'
 
-  gem.add_runtime_dependency "logging"
+  gem.add_runtime_dependency('dynflow', "~> 1.1")
+  gem.add_runtime_dependency('rest-client')
+  gem.add_runtime_dependency('sequel')
```

Comment:
This should be brought in by dynflow, iirc we don't use it directly anywhere else